lib/modules/manager/mise/extract.ts

```diff
@@ -79,26 +169,34 @@ function getConfigFromTooling(
   } // Return null if no toolDefinition is found
 
   return (
-    (typeof toolDefinition.config === 'function'
+    (is.function_(toolDefinition.config)
```

Comment:
```suggestion
    (is.function(toolDefinition.config)
```

we upgraded that dependency to a new major

Reply:
Thanks for the review!